### DISCUSSION TITLE
The view dashboard is not showing all the details in 5.0 version as it was showing in 4.6 version. After selecting the batch it is showing the details.

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#615] [General Discussion]
We are using the 4.6 Sunbird version and now we are upgrading to the 5.0 version
In 4.6 the batch gets selected by default but in 5.0 it asks to select the batch 
Is it a configuration change or a code change?
Can you guys help us find the solution here??
**This screenshot is of 4.6**
![image](https://github.com/Sunbird-Ed/Community/assets/109504832/6ca38322-d0a0-48ed-9b33-6c0d2d53fe4f)
**This screenshot is of 5.0**
![image](https://github.com/Sunbird-Ed/Community/assets/109504832/ded1e1a9-2f73-4f3b-b867-5743365f7c9d)
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by Krishnaj20** [comment]
@sachinb80 - Sorry - I'm not able to understand your query. What is the issue here?

**Reply by sachinb80** [reply]
@Krishnaj20 In the 4.6 version when we click on View dashboard button in any course, The batch is selected by default and all the details on that page is visible as you see in first screenshot But in 5.0 version the batch is not selected by default and all the details are not visible until we select the batch.

**Comment by sachinb80** [comment]
@Krishnaj20 Here i made one video of the question I have. Hope it helps

https://github.com/Sunbird-Ed/Community/assets/109504832/f1848435-2f98-4eef-b0a0-fc95fe1ad6a4

**Comment by Krishnaj20** [comment]
thanks @sachinb80 . After selecting the batch, is the dashboard working or you are facing any issues?

**Reply by sachinb80** [reply]
@Krishnaj20 After selecting the batch, the dashboard is working but my question is can we make like 4.6 version dashboard  by changing any configuration or code? Because the 4.6 version dashboard is looking more interactive and actractive.

**Reply by vinukumar-vs** [reply]
@ashokreddy1208 can you check this scenario from when this changes happened & what is the reason for this(any specific story).
@rajeevsathish

**Comment by ashokreddy1208** [comment]
@sachinb80 @vinukumar-vs Changes are done as part of material ui in 4.9 release, for now users has to select batch from dropdown to see course report for a batch.

cc @Krishnaj20 @rajeevsathish
